src/xdocs/config_blocks.xml

```diff
@@ -348,21 +348,36 @@ switch (a) {
           </tr>
           <tr>
             <td>exceptionVariableName</td>
-            <td>The name of variable associated with exception</td>
+            <td>
+              Specify the RegExp for the name of the variable associated with exception.
+              If Check meets variable name matching specified value - empty block is suppressed.
+            </td>
             <td><a href="property_types.html#regexp">Regular Expression</a></td>
-            <td>&quot;^$&quot; (empty)</td>
+            <td><code>&quot;^$&quot; (empty)</code></td>
             <td>6.4</td>
           </tr>
           <tr>
             <td>commentFormat</td>
-            <td>The format of the first comment inside empty catch</td>
+            <td>
+              Specify the RegExp for the first comment inside empty catch block.
+              If Check meets comment inside empty catch block matching specified format - empty
+              block is suppressed. If it is multi-line comment - only its first line is analyzed.
+            </td>
             <td><a href="property_types.html#regexp">Regular Expression</a></td>
-            <td>&quot;.*&quot;</td>
+            <td><code>&quot;.*&quot;</code></td>
             <td>6.4</td>
           </tr>
         </table>
       </subsection>
 
+      <subsection name="Notes" id="EmptyCatchBlock_Notes">
```

Comment:
This section looks out of place when the properties are already described first. By the time we get here, we already know about the properties and what they do.

Reply:
done